plugins/Model_GAN128/Model.py

```diff
@@ -42,7 +44,7 @@ class GANModel():
 
     def __init__(self, model_dir, gpus):
         self.model_dir = model_dir
-        self.gpus = gpus
+        self.gpus = 1 #See https://github.com/deepfakes/faceswap/issues/287

```

Comment:
I'd prefer you let the original line and put below it an assert, with a failure message pointing to the issue you mention in the comment